src/cmd/flux-wreck

```diff
@@ -141,6 +141,50 @@ prog:SubCommand {
  end
 }
 
+prog:SubCommand {
+ name = "exclude",
+ description = "Exclude a node from running future jobs",
+ usage = "NODENAME",
+ options = {
+   { name = "kill", char = "k",
+     usage = "kill the jobs to which the node is allocated"
+   }
+ },
+ handler = function (self, arg)
+    local function toboolean(X)
+         return not not X
+    end
+
+    local nodename = arg[1]
+    local resp, err = f:rpc ("sched.exclude", { node = nodename, kill = toboolean (self.opt.k) })
+    if not resp then
+        if err == "Function not implemented" and not self.opt.f then
+            prog:die ("Node exclusion is not supported when scheduler not loaded")
+        else
+            prog:die ("Unable to exclude node %s\n", nodename)
+        end
+    end
+ end
+}
+
+prog:SubCommand {
+ name = "include",
+ description = "Include a node for scheduling",
+ usage = "NODENAME",
+ handler = function (self, arg)
+    local nodename = arg[1]
+    local resp, err = f:rpc ("sched.include", { node = nodename })
+    if not resp then
+        if err == "Function not implemented" and not self.opt.f then
+            prog:die ("Node inclusion is not supported when scheduler not loaded")
```

Comment:
Same as above, is `and not self.opt.f` needed here?

Reply:
Thanks @grondo. I am not really a lua type. Will fix.